api/web3server.go

```diff
@@ -436,7 +436,12 @@ func (svr *Web3Server) sendRawTransaction(in interface{}) (interface{}, error) {
 	if err != nil {
 		return nil, err
 	}
-	tx, sig, pubkey, err := action.DecodeRawTx(dataStr, svr.coreService.EVMNetworkID())
+
+	chainiD, err := action.HandleChainID(dataStr, svr.coreService.EVMNetworkID())
+	if err != nil {
+		return nil, err
+	}
+	tx, sig, pubkey, err := action.DecodeRawTx(dataStr, chainiD)
```

Comment:
for chainID = 999999, the internal payload was actually native protobuf, so need to recover pubkey that way
see: https://github.com/iotexproject/iotex-antenna/blob/ledger-rlp/src/iotx.ts#L161-L178